### PR TITLE
Added install script

### LINKED ISSUE #185: Migrate ANR install process to use install script
Create an install script to allow users to install the ANR similar to avalanche-cli.

### PR DESCRIPTION
**Pull request by dasconnor**
Added script to match process similar to avalanche-cli.

Closes #185

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+PREFIX="$OWNER/$REPO"
+DEFAULT_INSTALL=~/bin
+
+usage() {
+  this=$1
+  cat <<EOF
+$this: download go binaries for $PREFIX
+
+Usage: $this [-b] bindir [-d] [tag]
+  -b sets bindir or installation directory, Defaults to $DEFAULT_INSTALL
```
Comment (holisticode) [inline scripts/install.sh:18]: Nit: Defaults lower case here

File: scripts/install.sh [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+# this function wraps all the destructive operations
+# if a curl|bash cuts off the end of the script due to
+# network, either nothing will happen or will syntax error
+# out preventing half-done work
+execute() {
+  tmpdir=$(mktemp -d)
+  log_debug "downloading files into ${tmpdir}"
+  http_download "${tmpdir}/${TARBALL}" "${TARBALL_URL}"
+  http_download "${tmpdir}/${CHECKSUM}" "${CHECKSUM_URL}"
+  hash_sha256_verify "${tmpdir}/${TARBALL}" "${tmpdir}/${CHECKSUM}"
```
Comment (holisticode) [inline scripts/install.sh:53]: very nice

File: scripts/install.sh [hunk 3]
```diff
@@ -0,0 +1,374 @@
...
+  rm -rf "${tmpdir}/${NAME}"
+  (cd "${tmpdir}" && untar "${TARBALL}")
+  test ! -d "${BINDIR}" && install -d "${BINDIR}"
+  for binexe in $BINARIES; do
+    if [ "$OS" = "windows" ]; then
+      binexe="${binexe}.exe"
```
Comment (holisticode) [inline scripts/install.sh:59]: Nit: in `get_binaries()` windows isn't supported anyways

**Review by holisticode — APPROVED** [review]
lgtm, a couple of nits at your discretion

File: scripts/install.sh (hunk 1 above)
Comment (dasconnor) [reply scripts/install.sh]: fixed

**Review by felipemadero — APPROVED** [review]
LGTM

**Pull request merged**